go/app/main.go

```diff
@@ -20,22 +24,116 @@ type Response struct {
 	Message string `json:"message"`
 }
 
+type Item struct {
+	ID 	     int    `json:"id"`
+	Name     string `json:"name"`
+	Category string `json:"category"`
+	Image    string `json:"image_name"`
+}
+
+type Items []Item
+
 func root(c echo.Context) error {
 	res := Response{Message: "Hello, world!"}
 	return c.JSON(http.StatusOK, res)
 }
 
 func addItem(c echo.Context) error {
 	// Get form data
+    id := c.FormValue("id")
+    c.Logger().Infof("Receive id: %s", id)
```

Comment:
[not covered in the text]
Indentation is not consistent here. It is good idea to consistently format the code in a team development.
Go has the default way to do so, please try

```
$ go fmt app/main.go
```

and see what happens.

Reply:
I usually use spaces instead of tabs, so there were some conflicts.
Is it natural for Golang users to indent by tabs rather than spaces?
If so, as a space user, do I have to run the `fmt` command every time before committing?

Reply:
That's the important point! In your personal project you can do whatever you want, but in a team development you should follow the team's way because consistency is crucial. And it is widely accepted in the Golang community that they use tabs for indentation, even though different opinions exist.
https://www.reddit.com/r/golang/comments/ee4dqn/how_to_make_gofmt_or_go_fmt_use_whitespaces/

At least, please don't mix tabs and spaces, or you'll end up like this 😄 

![image](https://github.com/F-0728/mercari-build-training/assets/486678/dfe24955-cc85-4670-b5b2-5935c01a4f76)